### PR TITLE
✨ Add upgrade link for widget v2

### PR DESCRIPTION
**Pull request by williamchong**
image

### CODE REVIEW

File: src/components/LikeCoinButtonWidgetV2/LikeCoinButtonWidgetV2.vue [hunk 1]
```diff
@@ -51,6 +51,23 @@
         <div :style="hintLabelStyle">{{ hintLabel }}</div>
       </foreignObject>
 
+      <!-- Upgrade -->
+      <foreignObject
+        v-if="upgradeHref"
+        :x="upgradeLabelX"
+        :y="upgradeLabelY"
+        width="60"
```
Comment (nwingt) [inline src/components/LikeCoinButtonWidgetV2/LikeCoinButtonWidgetV2.vue:59]: Is 60 enough for the word `upgrade`?

File: src/components/LikeCoinButtonWidgetV2/LikeCoinButtonWidgetV2.vue [hunk 2]
```diff
@@ -51,6 +51,23 @@
...
+        height="20"
+      >
+        <a
+          :href="upgradeHref"
+          target="_blank"
+          rel="noreferrer noopener"
+        >
+          <div :style="upgradeLabelStyle">{{ this.$t('HintLabel.Upgrade') }}</div>
+        </a>
```
Comment (nwingt) [inline src/components/LikeCoinButtonWidgetV2/LikeCoinButtonWidgetV2.vue:68]: W
```suggestion
          :style="upgradeLabelStyle"
        >{{ this.$t('HintLabel.Upgrade') }}</a>
```

File: src/components/LikeCoinButtonWidgetV2/LikeCoinButtonWidgetV2.vue [hunk 3]
```diff
@@ -192,6 +213,12 @@ export default {
         y: 130,
       };
     },
+    upgradeLabelX() {
+      return 198;
+    },
```
Comment (nwingt) [inline src/components/LikeCoinButtonWidgetV2/LikeCoinButtonWidgetV2.vue:218]: I think the x should align with avatar
image

File: src/components/LikeCoinButtonWidgetV2/LikeCoinButtonWidgetV2.vue (hunk 1 above)
Comment (nwingt) [reply src/components/LikeCoinButtonWidgetV2/LikeCoinButtonWidgetV2.vue:59]: Can be more allow it children to fill up space

File: src/components/LikeCoinButtonWidgetV2/LikeCoinButtonWidgetV2.vue [hunk 4]
```diff
@@ -233,6 +260,15 @@ export default {
         alignItems: 'flex-end',
       };
     },
+    upgradeLabelStyle() {
+      return {
+        ...this.textStyle,
+        width: '100%',
+        height: '100%',
+        display: 'flex',
```
Comment (nwingt) [inline src/components/LikeCoinButtonWidgetV2/LikeCoinButtonWidgetV2.vue:268]:
```suggestion
```

File: src/components/LikeCoinButtonWidgetV2/LikeCoinButtonWidgetV2.vue (hunk 3 above)
Comment (williamchong) [reply src/components/LikeCoinButtonWidgetV2/LikeCoinButtonWidgetV2.vue:218]: left align? seems too close to the like button/text though

Comment (williamchong) [reply src/components/LikeCoinButtonWidgetV2/LikeCoinButtonWidgetV2.vue:218]: image
How about this

Comment (nwingt) [reply src/components/LikeCoinButtonWidgetV2/LikeCoinButtonWidgetV2.vue]: How about `Upgrade`? Is it still align to the center?

Comment (nwingt) [reply src/components/LikeCoinButtonWidgetV2/LikeCoinButtonWidgetV2.vue]: If not, probably need to add `textAlign: center` to `upgradeLabelStyle` and add a fixed width

Comment (williamchong) [reply src/components/LikeCoinButtonWidgetV2/LikeCoinButtonWidgetV2.vue]: image
numbers should be align but visually seems 1 px off...

Comment (nwingt) [reply src/components/LikeCoinButtonWidgetV2/LikeCoinButtonWidgetV2.vue]: Probably due to the uppercase U, but I ll accept that

**Review by nwingt — APPROVED** [review]

**Pull request merged**